### ISSUE TITLE
Helper, no edit/delete for some helpers

### ISSUE THREAD

**Issue by averater** [home-assistant/frontend#17804]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

Same issue as https://github.com/home-assistant/frontend/issues/17375

For input_bolean, input_datetime, timer and input_select there is no edit or delete options.
Binary_sensor works as expected

Home Assistant 2023.8.4
Supervisor 2023.08.1
Frontend 20230802.1 - latest
No prompt or option to update so I guess this is latest.

Tested in companion app, Firefox, Firefox focus and duckduckgo on Android.

### Describe the behavior you expected

Option to edit and delete should appear. 

### Steps to reproduce the issue

1. Create helper
2. click on cogwheel

### What version of Home Assistant Core has the issue?

2023.8.4 

### In which browser are you experiencing the issue with?

Firefox 117.0

### Which operating system are you using to run this browser?

android 13

**Comment by karwosts** [comment]
Could you elaborate a bit more what you mean "there is no delete option". Do you see anything at all?

A screenshot would be helpful.

I have very rarely seen the settings menu be missing when I click the cogwheel, but it always fixes itself after a page refresh. Is that like what you're seeing, and do you see it consistently?

**Comment by averater** [comment]
I just updates code plus frontend and in companion app i could see all config. But closed it and opened and it was gone. Tried closing the app and restart it but still gone.

I now have:
Home Assistant 2023.9.0
Supervisor 2023.08.3
Frontend 20230906.1 - latest
App: 2023.8.2-full

It looks as the described in the linked ticket. Ony the header visible, the rest is completely blank.

**Comment by averater** [comment]
![Screenshot_20230907_082926_Home Assistant](https://github.com/home-assistant/frontend/assets/20190223/17b86dd1-2a92-46ac-aec9-6d1d5b1e437a)
![Screenshot_20230907_082940_Home Assistant](https://github.com/home-assistant/frontend/assets/20190223/30c60417-6f8e-4648-a1cd-35848f0558a7)
![Screenshot_20230907_082948_Home Assistant](https://github.com/home-assistant/frontend/assets/20190223/cc69243c-77e7-45fb-92c1-4a269005c980)
![Screenshot_20230907_082957_Home Assistant](https://github.com/home-assistant/frontend/assets/20190223/3e45c1df-8ac2-4d25-acfb-b7d3372b3d28)
![Screenshot_20230907_083002_Home Assistant](https://github.com/home-assistant/frontend/assets/20190223/cb5fc7ec-1671-4904-9a3a-d9fa27306a42)

Tried closing the app, reboot phone and start app again, still no config.

Here are some screens hots, one of my helper list. The top helper is configured in yaml so no edit of that one. Second Akvariebelysning works as expected, third Akvarietimer has no config.

**Comment by averater** [comment]
Is this relevant?  Noticed in system log:

Logger: homeassistant.helpers.template
Source: helpers/template.py:239
First occurred: 06:41:20 (10 occurrences)
Last logged: 12:51:50

Template variable warning: 'dict object' has no attribute 'command' when rendering '{{ trigger.event.data.command }}'
Template variable warning: 'dict object' has no attribute 'cluster_id' when rendering '{{ trigger.event.data.cluster_id }}'
Template variable warning: 'dict object' has no attribute 'endpoint_id' when rendering '{{ trigger.event.data.endpoint_id }}'
Template variable warning: 'dict object' has no attribute 'args' when rendering '{{ trigger.event.data.args }}'

**Comment by averater** [comment]
Different computer, got it working.
Ubuntu 22, chromium incognito.

**Comment by dsellers1** [comment]
TL;DR: Resolved in 2023.9.1

I read the first message and thought to myself "I was able to delete a helper a couple of weeks ago. It works." But, sure enough, I had the same symptoms as the OP in Firefox 117.0 and HA 2023.9.0. Worked fine in Edge. I did not grab screenshots at the time. Updated to 2023.9.1 and the issue is now gone. Clicking the cog wheel brings up the settings like it is supposed to.

**Comment by averater** [comment]
Not resolved for me. I only get it to work on one computer and one browser. Not on my Android phone with any browser nor companion app. My other computer also not working with and browser. Both computers have Ubuntu 22. Will check browser versions.

**Comment by karwosts** [comment]
Yeah I don't think anything went into 9.1 that would fix this. I think this has been an issue for a while, though it seems to happen only sporadically to some users.

**Comment by koenkv** [comment]
I have the same problem is there already a fix for?

**Comment by lionslair** [comment]
Is this ever going to get looked at. Had helpers I can not delete for months and the open new tab thing did nothing for me
